src/state-mutation.js

```diff
@@ -12,14 +12,16 @@ const updateReverseRelationship = (
   return (foreignEntities) => {
     return foreignEntities.update(
       foreignEntities.findIndex(
-        item => item.get('id') === relationship.getIn(['data', 'id'])
+        item => item.get('id') == relationship.getIn(['data', 'id'])
```

Comment:
Considering that the API will provide all ID's, I would think that it's fair to assume that they are of same object type for the same resource type. Don't know if they would vary between resource types, but an API might have some resources with incremental ID's and others with UUID's.